examples/helm/templates/jvb/service.yaml

```
@@ -0,0 +1,34 @@
+{{- if default (not .Values.jvb.useHostPort) .Values.jvb.service.enabled }}
```

Setting  `jvb.service.enabled` to `false` doesn't work as I expected : the test is evaluated as `true` and the resource is still created.

I think there's some confusion around `default` usage, see for example this [Helm issue](https://github.com/helm/helm/issues/3308#issuecomment-585349501).

A workaround bellow (doesn't handle the case of `jvb.service` set to `null` though)

```suggestion
{{- if and .Values.jvb.service.enabled (not .Values.jvb.useHostPort) }}
```

---

Argggg, stupid helm documentation, https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function. This is hugely misleading. So now I believe that all the default statement are wrong. Will check that soon